playbooks/openshift-glusterfs/private/add_hosts.yml

```diff
@@ -0,0 +1,12 @@
+---
+# This play runs when new gluster hosts are part of new_nodes group during
+# master or node scaleup.
```

Comment:
i guess you never want in prod to run during master scaleup, no ?

Reply:
I'm not sure what you mean.  This would run during master scaleup, yes.  It's up to the user if they decide to make masters also glusterfs hosts.